### PR TITLE
Create a basic backend structure

### LINKED ISSUE #1: Create a backend image
Create a backend folder in the Loxotron repo and put basic sources and Dockerfile in there

### PR DESCRIPTION
**Pull request by dkushche**

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+const app = express()
+
+app.get('/*', (req, res) =>{
+    
+    res.status(200).json('Hello world!')
+
+})
+
+app.listen(PORT, () => console.log("SERVER STARTED ON PORT " +PORT ))
```
Comment (dkushche) [inline backend/index.js]: @stas3729 Add newline in the end of the file

**Comment by dkushche** [comment]
Add node_modules directory into .gitignore

File: backend/index.js (hunk 1 above)
Comment (dkushche) [reply backend/index.js]: Here how good code style looks like
`console.log("SERVER STARTED ON PORT " + PORT)`

File: backend/index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import express from 'express'
+
+const PORT  = 5000;
+
```
Comment (dkushche) [inline backend/index.js]: redundant newline

**Review by skravchenk — APPROVED** [review]

**Pull request merged**